deployment-guide/upgrade-guides/upgrading-cinchy-versions/kubernetes-upgrades/v5.7-kubernetes.md

```diff
@@ -0,0 +1,203 @@
+
+{% hint style="success" %}
+You may have changed the name of the `deployment.json` file during your original platform deployment. If so, make sure that you swap up the name wherever it appears in this document.
+{% endhint %}
+
+### Remove components
+
+In the 5.7 templates, the cluster-level components will upgrade to the latest version. You need to remove `kube-prometheus-stack`, `logging-operator app `and `kafka-cluster (Warning)` from ArgoCD. This change deletes your recent metrics from Grafana and you will only see the latest metrics after you deploy the new `kube-prometheus-stack`. The older CRDs created by `kube-prometheus-stack `and logging-operator charts aren't removed by default during upgrade and should be manually cleaned up with the below commands:
+
+```bash
+kubectl delete crd alertmanagerconfigs.monitoring.coreos.com alertmanagers.monitoring.coreos.com podmonitors.monitoring.coreos.com probes.monitoring.coreos.com prometheusagents.monitoring.coreos.com prometheuses.monitoring.coreos.com prometheusrules.monitoring.coreos.com scrapeconfigs.monitoring.coreos.com servicemonitors.monitoring.coreos.com thanosrulers.monitoring.coreos.com
+```
+
+```bash
+kubectl delete crd clusterflows.logging.banzaicloud.io clusteroutputs.logging.banzaicloud.io flows.logging.banzaicloud.io loggings.logging.banzaicloud.io outputs.logging.banzaicloud.io eventtailers.logging-extensions.banzaicloud.io fluentbitagents.logging.banzaicloud.io hosttailers.logging-extensions.banzaicloud.io nodeagents.logging.banzaicloud.io syslogngclusterflows.logging.banzaicloud.io syslogngclusterflows.logging.banzaicloud.io syslogngclusteroutputs.logging.banzaicloud.io syslogngflows.logging.banzaicloud.io syslogngoutputs.logging.banzaicloud.io
+```
+
+### Redeploy components
+
+After the CRDs are deleted then `redeploy logging-operator` and `kube-prometheus-stack` app with below command. Change directory to **cinchy.argocd** repository:
+
+```bash
+bash deploy_cluster_components.sh
+```
+
+### Sync ArgoCD apps
+
+You may see a couple of ArgoCD apps out of sync that are because you had deleted the logging operator. Please sync them manually.
+
+1. Open a shell/terminal from the cinchy.devops.automations directory and execute the following command:
+   ```bash
+   dotnet Cinchy.DevOps.Automations.dll "deployment.json"
+   ```
+1. Commit all of your changes (if there were any) in each repository.
+1. If there were any changes in your **cinchy.argocd** repository you may need to redeploy ArgoCD.Launch a shell/terminal with the working directory set to the root of the **cinchy.argocd** repository.
+1. Execute the following command to deploy ArgoCD:
+   ```bash
+   bash deploy_argocd.sh
+   ```
+1. Execute the following command from the **cinchy.argocd** repository:
```

Comment:
This is verbatim from the email sent @jadhavnitind. I'm not sure if there's a missing item here, as the colon seems to indicate there should be code below. Not sure if it's a typo.

Reply:
Please remove this line.